spec/support/shared_examples/a_protected_zendesk_ticket_page.rb

```diff
@@ -0,0 +1,40 @@
+shared_examples :a_protected_zendesk_ticket_page do |action: :show|
```

Comment:
interesting syntax there, I didn't realize you could have a block argument with a default value...

Reply:
Also worth noting: I used a symbol to define the shared example because I get frustrated when I can't easily navigate between shared examples and the specs that use them. By making it a symbol, RubyMine is able to quickly identify all usages of the shared example.